modules/docs/src/main/mdoc/docs/03-Connecting.md

```diff
@@ -46,6 +46,23 @@ val xa = Transactor.fromDriverManager[IO](
 )
 ```
 
+To use sqlite, your code like 
```

Comment:
Thanks for the contribution :) Since there are no specific integration with sqlite I'd like to avoid adding too much info about it, as all that's different is the driver class name and the connection URL.

May I suggest that we change all of this to something like the following to be more generic and concise?

```
Note: If you're connection to anyother databases , make sure to change the JDBC driver class name and connection URL according to their documentation. 
```

Reply:
I total agree with you. 

It may be my specific problem. I PR this because I never learn(in deeper way) java language. So it's very difficult for me to find the solution for sqlite.

Please close the PR.